tooling/create-extension-templates/runtime-pom.xml

```diff
@@ -33,7 +33,7 @@
     <description>[=description]</description>
 
     <properties>
-        <firstVersion>1.0.0-M8</firstVersion>
+        <firstVersion>1.0.0-CR2</firstVersion>
```

Comment:
Should it not be CR1 ?

Reply:
No it  is correct. That's the template for newly created extensions, which will effectively appear in the n+1 release.